### DISCUSSION TITLE
.env laguage

### DISCUSSION THREAD

**Discussion by SMAV12221** [n3d1117/chatgpt-telegram-bot#308] [Q&A] [answered]
![image](https://github.com/n3d1117/chatgpt-telegram-bot/assets/116415774/2430a96a-bade-4339-a074-fc88be5e5f8e)
![image](https://github.com/n3d1117/chatgpt-telegram-bot/assets/116415774/9c859a47-9cb6-4686-8d06-813ec49c2df5)
I changed the language in the .env file, but the bot itself does not change the language, how to fix it?

**Comment by n3d1117** [comment, marked as answer]
Hey there, did you remove the `#` before `BOT_LANGUAGE`? And have you restarted the bot?

**Reply by SMAV12221** [reply]
Thank's
